.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "git.ignoreLimitWarning": true
+}
```

Comment:
Looks like you are missing a .gitignore file which is why you are having to put this here and also why your .env was committed.